### DISCUSSION TITLE
Where were you ML capstone submission leaderboard error

### DISCUSSION THREAD

**Discussion by audichandra** [teamalgoritma/community#799] [Q&A] [answered]
saya coba untuk submit hasil prediction image classification where were you ke algoritma leaderboard dengan format yang sama dengan submission example tapi yang keluar malah error  (saya sudah coba dengan row.names = FALSE juga ndak bisa) 
![errorcap](https://github.com/teamalgoritma/community/assets/45793211/6cb2f367-285b-4b7e-a44d-f8ac482ce4ad)
![submission](https://github.com/teamalgoritma/community/assets/45793211/8d4f41d8-70ad-4bf6-816c-8656d2b6904d)

**Comment by IrfanChairurrachman** [comment]
Dear @audichandra  ,

Pertanyaan Anda sudah kami baca dan sedang kami telusuri permasalahannya. Harap menunggu untuk balasan selanjutnya, terima kasih banyak.

**Comment by kinantytasyao** [comment, marked as answer]
Dear @audichandra,

Kami telah menyelesaikan error yang Bapak hadapi dalam melakukan submission, Silakan dicoba lagi ya leaderboardnya. Semoga membantu :)

Mark as answer jawaban ini jika Anda telah berhasil mengecek submission pada leaderboard✅

**Reply by IrfanChairurrachman** [reply]
Halo @audichandra , untuk menjaga agar forum diskusi berjalan kondusif, kami izin menandai pertanyaan "mark as answer" dan melakukan close pada diskusi ini karena belum ada respon setelah 7 hari respon terakhir. Semoga dapat dipahami. Terima kasih.

**Comment by vinsensiaellen** [comment]
Selamat pagi, saya masih belum mendapatkan akses untuk login ke leaderboard, bisa tlg dibantu, terimakasih:)

**Comment by kinantytasyao** [comment]
Dear @vinsensiaellen,

Berikut adalah username dan password dari leaderboard milik Anda:

- Username: ellensetiawan
- Password: PzGR

Semoga membantu :)

**Comment by fransiskajp** [comment]
Selamat sore, saya belum mendapatkan akses untuk login ke leaderboard, untuk username dan password nya apa yah? Terima kasih

**Reply by kinantytasyao** [reply]
Dear @fransiskajp,

Berikut adalah username dan password dari leaderboard milik Anda:
- Username: fransiskajesintapinem
- Password: fnAF

Semoga membantu :)
